File: src/main/java/com/example/interviewlandbackend/dto/request/CreateSectionRequest.java
package com.example.interviewlandbackend.dto.request;

import javax.validation.constraints.NotBlank;
import javax.validation.constraints.NotNull;

public class CreateSectionRequest {

    @NotBlank
    private String sectionName;

    @NotNull
    private int contentId;


    public String getSectionName() {
        return sectionName;
    }

    public int getContentId() {
        return contentId;
    }



}

File: src/main/java/com/example/interviewlandbackend/dto/request/CreateSectionRequest.kt
package com.example.interviewlandbackend.dto.request

import javax.validation.constraints.NotBlank
import javax.validation.constraints.NotNull

data class CreateSectionRequest @JvmOverloads constructor(

    @field:NotBlank
    val sectionName: String,

    @field:NotNull
    val contentId: Int


) {
}